filebeat/module/apache/access/ingest/pipeline.yml

```diff
@@ -14,7 +14,7 @@ processors:
     - '%{IPORHOST:source.address} - %{DATA:user.name} \[%{HTTPDATE:apache.access.time}\]
       "(?:%{WORD:http.request.method} %{DATA:url.original} HTTP/%{NUMBER:http.version}|-)?"
       %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)(
-      "%{DATA:http.request.referrer}")?( "%{DATA:user_agent.original}")?'
+      "%{DATA:http.request.referrer}")? ("%{DATA:user_agent.original}")?'
```

Comment:
Umm, this pattern is not equivalent. After this change an space after the referrer will be mandatory, is this expected?